addons/crm/tests/test_crm_lead.py

```diff
@@ -119,6 +119,36 @@ def test_crm_lead_creation_partner(self):
         self.assertEqual(lead.zip, self.contact_1.zip)
         self.assertEqual(lead.country_id, self.contact_1.country_id)
 
+    @users('user_sales_manager')
+    def test_crm_lead_suggested_recipient_partner_no_email(self):
```

Comment:
There is a 'test_lead_message_get_suggested_recipient' test in test_lead_notification.py file, I think we should improve that test instead of adding another one